test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -405,5 +420,16 @@ private static void VerifyFile(string file, string content)
                     break;
             }
         }
+
+        private static void VerifyLogsHasLineInfo(string[] logs)
+        {
+            foreach (var log in Array.ConvertAll(logs, JArray.Parse))
+            {
+                if (!s_errorCodesWithoutLineInfo.Contains(log[1].ToString()) && log.Count < 6)
+                {
+                    Assert.True(false, $"Error code {log[1].ToString()} must have line info");
```

Comment:
that's means by default newly added error should have line info, unless we add it to the exclusion list?

Reply:
yep!